common/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "common"
```

Comment:
what is the purpose of this going to be? To share types?

Reply:
Common types, shared functions, error handling, database connection strings, secrets, database schema

Reply:
Common types make sense, but anything DB related will be a BE thing